### DISCUSSION TITLE
Suggestion on page /foundations/why-python.html

### DISCUSSION THREAD

**Discussion by kitmacleod** [ProjectPythia/pythia-foundations#152] [Ideas]
A key page if coming from Zero Python knowledge (or from a small amount of R, Matlib etc knowledge). Maybe split into two: what is Python and why Python (I appreciate these are linked). I would include Python information like https://www.python.org/about/gettingstarted/, Python (like Go) is 'batteries included' (still need external packages, not to the same extent at Node.js) and link to PyPi (has had a huge investment over recent years, as sometimes considered a weakness of the Python ecosystem compared to other languages). Highlight Python's strengths (e.g. often taught as the first CS language and lots of enterprise Python jobs) impt but also mention the strengths of other key language communities e.g. R and Julia). I would include other key and recent Python learning resources (books and YouTube, ideally more recent that Traversy 2018 Crash course or Microsoft's 2019 video series; as an aside like Fireship.io x in 100 seconds, but not available for Python).

**Comment by brian-rose** [comment]
Thanks @kitmacleod for these comments! @ProjectPythia/education we didn't get to this one today but we'll discuss at a future EWG meeting.

**Comment by brian-rose** [comment]
Migrated this over to the Discussions page for ... discussion!
